lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -905,16 +905,18 @@ public function get_collection_params() {
 		$query_params['context']['default'] = 'view';
 
 		$query_params['exclude'] = array(
-			'description'        => __( 'Ensure result set excludes specific ids.' ),
-			'type'               => 'array',
-			'default'            => array(),
-			'sanitize_callback'  => 'wp_parse_id_list',
+			'description'       => __( 'Ensure result set excludes specific ids.' ),
+			'type'              => 'array',
+			'default'           => array(),
+			'sanitize_callback' => 'wp_parse_id_list',
+			'validate_callback' => 'rest_validate_request_arg',
 		);
```

Comment:
The `items` schema needs to be added into all of the array parameters so that your validation is actually working.  This is why all of the tests are failing because the schema is not set yet.  Every include/exclude author_include/exclude parameter needs it and I believe user roles.  There are some other ones that I can't remember.